akka-typed/src/main/scala/akka/typed/scaladsl/Actor.scala

```diff
@@ -286,41 +286,45 @@ object Actor {
    * system to reuse the previous behavior. This is provided in order to
    * avoid the allocation overhead of recreating the current behavior where
    * that is not necessary.
+   *
+   * Same as [[Behavior.Same]], but replicated here for convenienve when using the the Actor DSL via import `Actor._`.
    */
-  def Same[T]: Behavior[T] = SameBehavior.asInstanceOf[Behavior[T]]
+  def Same[T]: Behavior[T] = Behavior.Same
 
   /**
    * Return this behavior from message processing in order to advise the
    * system to reuse the previous behavior, including the hint that the
    * message has not been handled. This hint may be used by composite
    * behaviors that delegate (partial) handling to other behaviors.
+   *
+   * Same as [[Behavior.Unhandled]], but replicated here for convenienve when using the the Actor DSL via import `Actor._`.
    */
-  def Unhandled[T]: Behavior[T] = UnhandledBehavior.asInstanceOf[Behavior[T]]
-
-  /*
-   * TODO write a Behavior that waits for all child actors to stop and then
-   * runs some cleanup before stopping. The factory for this behavior should
-   * stop and watch all children to get the process started.
-   */
+  def Unhandled[T]: Behavior[T] = Behavior.Unhandled
 
   /**
    * Return this behavior from message processing to signal that this actor
    * shall terminate voluntarily. If this actor has created child actors then
    * these will be stopped as part of the shutdown procedure. The PostStop
    * signal that results from stopping this actor will NOT be passed to the
    * current behavior, it will be effectively ignored.
+   *
+   * Same as [[Behavior.Stopped]], but replicated here for convenienve when using the the Actor DSL via import `Actor._`.
    */
-  def Stopped[T]: Behavior[T] = StoppedBehavior.asInstanceOf[Behavior[T]]
+  def Stopped[T]: Behavior[T] = Behavior.Stopped
 
   /**
    * A behavior that treats every incoming message as unhandled.
+   *
+   * Same as [[Behavior.Empty]], but replicated here for convenienve when using the the Actor DSL via import `Actor._`.
    */
-  def Empty[T]: Behavior[T] = EmptyBehavior.asInstanceOf[Behavior[T]]
+  def Empty[T]: Behavior[T] = Behavior.Empty
 
   /**
    * A behavior that ignores every incoming message and returns “same”.
+   *
+   * Same as [[Behavior.Ignore]], but replicated here for convenienve when using the the Actor DSL via import `Actor._`.
    */
-  def Ignore[T]: Behavior[T] = IgnoreBehavior.asInstanceOf[Behavior[T]]
+  def Ignore[T]: Behavior[T] = Behavior.Ignore
 
   /**
```

Comment:
This is actually somewhat weird sadly... there's A TON of behaviors in Actor, like `static public <T> Behavior<T> tap(Procedure2<ActorContext<T>, Signal> signal, Procedure2<ActorContext<T>, T> message,
      Behavior<T> behavior) {` and `Monitor` and more.
Some are exposed as raw `case class`es defined in Actor - I'm not sure it's the best place to keep accumulating all useful behaviors. I understand the fancy-ness of "one-import actors", but it'll feel weird for other libraries to provide Behaviors. If we only do `Bevarior....` then others would do `MyBehavior....` and it feels more consistent.

Anyway, just raising the point that there is a lot of behaviors in here, and I'm not so sure anymore about just moving a few out of here. (While I would indeed really want to move them, reads more natural - it's not an Actor.Monitor, it's a Behavior that "monitors" etc.).